pytripgui/canvas_vc/plotter/managers/voi_manager.py

```diff
@@ -182,8 +182,10 @@ def _remove_all_voi_plots(self, name: str) -> None:
 
     @staticmethod
     def _get_color(voi: Voi) -> [float, float, float]:
-        # color in voi is stored as [a, b, c] where a,b,c are floats from 0.0 to 1.0
-        return voi.get_color()
+        # color in voi is stored as [a, b, c] where a,b,c are numbers from 0 to 255
+        # dividing by 255.0 to have float values from 0.0 to 1.0 to adapt to matplotlib colors
+        color = np.array(voi.get_color()) / 255.0

```

Comment:
Do we really need numpy array for 3-element list ?
What about 

```color = [color/255.0 for color in voi.get_color() ]```

We could also consider making color a tuple instead of list

Reply:
Done